style.css

```diff
@@ -0,0 +1,167 @@
+*{
+    margin:0;
+    padding: 0;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
Please put the font family in your the body

Reply:
When the font family is in the body tag is applies to all element in your HTML body which is a best practice.